lib/Api/api.dart

```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-import 'package:fluttertoast/fluttertoast.dart';
```

Comment:
api: Remove toast dependency
As we have integrated a hardware scanner and this application will now run as a desktop application we no longer need a mobile toast.